lib/lang/countries_lang_codes_table.dart

```diff
@@ -29,10 +28,10 @@ class CountriesLangCodesTable {
 
 // Source: https://wiki.openstreetmap.org/wiki/Nominatim/Country_Codes
 const _COUNTRIES_LANG_CODES_TABLE = {
-  Country.ANDORRA: ['ca'],
```

Comment:
Because i changed country to a buildvalue class. And maybe i am missing something, but that means i have to change all the countries keys to a Country. And change the lookup method also to use Country and not String. I created a task to modify it in github